libraries/commerce/favorites/streams/index.js

```diff
@@ -40,6 +44,6 @@ export const favoritesDidUpdate$ = main$.filter(({ action }) => [
 ].includes(action.type));
 
 export const favoritesWillRemoveItem$ = main$
-  .filter(({ action }) => action.type === REQUEST_REMOVE_FAVORITES);
+  .filter(({ action }) => action.type === REQUEST_REMOVE_FAVORITES && !action.silent);
```

Comment:
This looks like a dirty hack just to fulfil a very specific use case. What if any extension listens for this stream just to update it's view or to do something on the actually available entries only.
In fact this can be considered a breaking change of the public api (and therefore would require a major version bump).